deploy/azure/cloudbeat-azure.yml

```diff
@@ -0,0 +1,58 @@
+cloudbeat:
+  config:
+    v1:
+      benchmark: cis_azure
+  type: cloudbeat/cis_azure
+  # Defines how often an event is sent to the output
+  period: 30s
```

Comment:
Will we use Azure credentials, and should we configure them?

Reply:
Probably, but let's track this in https://github.com/elastic/cloudbeat/pull/1308

Reply:
thanks, cloudbeat-azure.yml LGTM